services/server/server_test.go

```diff
@@ -35,6 +35,10 @@ import (
 	"github.com/percona/pmm-managed/utils/tests"
 )
 
+type checkerMock struct{}
```

Comment:
It's better to generate mock using mockery like here https://github.com/percona/pmm-managed/pull/360/files#diff-60d6255f64d5cf44c23f4f2361cc1a30R29